submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,107 @@
+## A scientific examination of the Factorio world
+
+Suspension of disbelief is a well-known concept, however that does not stop me from ignoring it completely. Some days ago I saw a [Reddit post](https://www.reddit.com/r/factorio/comments/jaz6yl/calculating_the_density_of_nauvis/) about calculating the density of Nauvis, and that gave me an idea of making a just-for-fun compilation of all the ridiculous science that goes on in the Factorio universe.
+
+Obvious warning: This article contains excessive amounts of math, if you have a crippling math allergy, feel free to skip it.
+
+### Nauvis is a white dwarf
+
+This is a retelling of the Reddit post by [u/DaveMcW](https://www.reddit.com/user/DaveMcW/) that I just mentioned. Note that this assumes Nauvis's day/night cycle is actually caused by rotation, which is something I'll be arguing against later on.
+
+For a planet to not rip itself apart, gravitational acceleration must be greater than centripetal acceleration:
```

Comment:
Maybe add a description of why this is (ideally in like 1 sentence) as it is the main setup of this physics method. Not sure exactly what that would look like and if you cant think of anything don't worry.